src/legacy/core_plugins/console/server/proxy_route.js

```diff
@@ -105,42 +103,54 @@ export const createProxyRoute = ({
       const { path, method } = query;
       const uri = resolveUri(baseUrl, path);
 
+      // Because this can technically be provided by a settings-defined proxy config, we need to
+      // preserve these property names to maintain BWC.
       const { timeout, rejectUnauthorized, agent, headers } = getConfigForReq(req, uri);
```

Comment:
Note that `rejectUnauthorized` and `agent` are currently not being consumed. We should verify what kind of behavior depended upon them originally and ensure that behavior still exists.